tldw_Server_API/app/api/v1/endpoints/character_chat_sessions.py

```
@@ -721,31 +758,99 @@ async def _offline_sse():
+                            # Ensure DONE if provider didn't send one
+                            await stream.done()
+                        except Exception as e:
+                            await stream.error("internal_error", f"{e}")
+
+                    async def _generator():
+                        producer = asyncio.create_task(_produce_async())
+                        try:
+                            async for line in stream.iter_sse():
+                                yield line
+                        except asyncio.CancelledError:
+                            raise
```

Ruff (0.14.3)

Remove exception handler; error is immediately re-raised

(TRY203)